src/datachain/lib/file.py

```diff
@@ -195,14 +195,15 @@ def open(self, mode: Literal["rb", "r"] = "rb"):
             with VFileRegistry.resolve(self, self.location) as f:  # type: ignore[arg-type]
                 yield f
```

Comment:
Nitpicky suggestion: Instead of `else`, you can early return from `if` branch, and keep the rest of the code same.


But no strong opinion.

```suggestion
                yield f
                return
```